<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogLevelPanel</class>
 <widget class="QWidget" name="LogLevelPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogLevelPanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Selection</string>
     </property>
     <widget class="QPushButton" name="selectAll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Select All</string>
      </property>
     </widget>
     <widget class="QPushButton" name="unselectAll">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>92</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Unselect All</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>Finest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Finer</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fine</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Config</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Info</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Warning</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Severe</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply to all selected</string>
      </property>
     </widget>
     <widget class="QPushButton" name="applyBt">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>53</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="sourceWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>447</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="sourceWidgetLayou" stretch="">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="margin">
        <number>1</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>setDiscardLevel(QString)</slot>
 </slots>
</ui>
